src/zenml/services/service_monitor.py

```diff
@@ -15,7 +15,7 @@
 from abc import abstractmethod
 from typing import TYPE_CHECKING, Optional, Tuple
 
```

Comment:
Not sure, where to put this but as I was reading through the files, I have realized that there is an unused `error` variable at line 137 of this file.